packages/AlgernonTests.package/ALGTest.class/instance/nameOfFirstResult.st

```diff
@@ -1,3 +0,0 @@
-helpers
-nameOfFirstResult
```

Comment:
The Removal of this file creates the errors in the tests. Was there a reason you removed it ?

Reply:
no - must have been some merge errors as I initially branched from a old dev branch...